.github/workflows/ci-wheels.yml

```diff
@@ -0,0 +1,120 @@
+# Reference:
+#   - https://github.com/actions/checkout
+#   - https://github.com/actions/download-artifact
+#   - https://github.com/actions/upload-artifact
+#   - https://github.com/pypa/cibuildwheel
+#   - https://github.com/pypa/build
+#   - https://github.com/pypa/gh-action-pypi-publish
+#   - https://test.pypi.org/help/#apitoken
+
+name: ci-wheels
+
+on:
+  pull_request:
+
+  push:
+    branches:
+      - "main"
+      - "v*x"
+      - "!auto-update-lockfiles"
+      - "!pre-commit-ci-update-config"
+      - "!dependabot/*"
+    tags:
+      - "v*"
+
+jobs:
+  build_bdist:
+    name: "build ${{ matrix.os }} (${{ matrix.arch }}) wheels"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["ubuntu-latest", "macos-latest", "windows-latest"]
+        arch: ["x86_64", "arm64"]
+        exclude:
+          - os: "ubuntu-latest"
+            arch: "arm64"
+          - os: "windows-latest"
+            arch: "x86_64"
+          - os: "windows-latest"
+            arch: "arm64"
+        include:
+          - os: "windows-latest"
+            arch: "AMD64"
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+  build_sdist:
+    name: "Build sdist"
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - name: "Building sdist"
+      shell: bash
+      run: |
+        pipx run build --sdist 
+
+    - uses: actions/upload-artifact@v4
+      with:
+        name: pypi-sdist
+        path: ${{ github.workspace }}/dist/*.tar.gz
+
+
+  show-artifacts:
```

Comment:
Several other repos include a test stage. Looks like they download the artifacts and make sure that each provides a functional instance of the package.

Repos with this:

- Iris
- Iris-esmf-regrid
- nc-time-axis

Repos without this:

- Stratify
- cf-units

It looks like the only times that we skip this are for repos that include Cython (and therefore build using cibuildwheel). Tephi is not in this camp, so could it include a testing stage?